### DISCUSSION TITLE
using Docker / providing images

### DISCUSSION THREAD

**Discussion by andy5995** [dosemu2/dosemu2#1489] [Ideas]
In https://github.com/dosemu2/dosemu2/issues/1485#issuecomment-852533366 @stsp said,

>Btw we support make deb and make rpm
to create packages locally.
Maybe you can integrate also
make docker into our build system?

I'm new to Docker and still learning.

I haven't seen any projects yet with a `make docker` option and not sure what the goal of that would be. What would you want the image to contain? You said binaries already exist?

You maybe would like a docker image of just the dosemu2 binary without its build dependencies? (I saw some on the docker hub but I think they might be outdated.)

Today I learned something that would make the docker image I created more beneficial to use for developers.

First of all, the cloned git repo could be removed from the image.

A developer could clone the dosemu2k repo locally, then run docker like this (while in the directory of the local repo).
`docker run -v $PWD:/dosemu2 andy5995/dosemu2-dev`

That would mount the local repo inside the image at /dosemu2. From there, they could run ./autogen.sh, etc.

While the container is running, they can edit the files locally and rebuild inside the container to test changes.

**Comment by stsp** [comment]
Yes, I have no idea why do we
need a docker image with dosemu2. :)
Which is why I asked you to tell
us why do we need that:
```
It should contain the detailed description
of what is it, how to use and who needs
that.
```

> You maybe would like a docker image of just the dosemu2 binary without its build dependencies?

I would imagine the build deps
are not needed there. But its hard
to tell before I know why you do
this all. :)

**Comment by andy5995** [comment]
I need to get more experience with docker. I've removed the image and the docker info from the wiki.
